Src/Microsoft.Dynamic/Runtime/BinderOps.cs

```diff
@@ -152,6 +152,10 @@ public static class BinderOps {
             return new ArgumentTypeException($"{name}() got multiple values for keyword argument '{argumentName}'");
         }
 
+        public static ArgumentTypeException TypeErrorForDuplicateArgument(string name, int position, string argumentName) {
+            return new ArgumentTypeException($"Argument for {name}() given by name ('{argumentName}') and position ({position})");
```

Comment:
If we're picking the message to match CPython, then in newer versions it looks like they use lower-case "a" in argument. I don't particularly care either way, just something to consider.